src/main/resources/templates/courseFormView.twig

```diff
@@ -0,0 +1,50 @@
+{% extends 'base.twig' %}
+
+{% block title %}
+    <h1>Course form goes here</h1>
+{% endblock %}
+
+{% block body %}
+    <script type="text/javascript">
+        $(document).ready(function() {
+            $('.datepicker').pickadate({
+                formatSubmit: 'yyyy-mm-dd'
+            });
+            $('.timepicker').pickatime({
+                twelvehour: true
+            });
+        });
+    </script>
+
+    <div style="width: 40%">
+        <form action="/course" method="post">
+            <div class="md-form">
+                <input type="text" name="courseName" class="form-control" id="courseFormName">
```

Comment:
This name will have to be changed

Reply:
Just for clarification, what needs to be renamed?

Reply:
`name=courseName` needs to be changed to `name=name`